fabric_rti_mcp/eventstream/eventstream_service.py

```diff
@@ -0,0 +1,357 @@
+"""
+Eventstream service module for Microsoft Fabric RTI MCP
+Provides Microsoft Fabric Eventstream operations through MCP tools
+Uses Azure Identity for transparent authentication (consistent with Kusto module)
+"""
+
+import asyncio
+import base64
+import json
+import os
+import uuid
+from datetime import datetime
+from typing import Any, Dict, List, Optional, Coroutine
+
+from fabric_rti_mcp.common import logger
+from fabric_rti_mcp.eventstream.eventstream_connection import EventstreamConnection
+
+# Microsoft Fabric API configuration
+DEFAULT_FABRIC_API_BASE = "https://api.fabric.microsoft.com/v1"
+DEFAULT_TIMEOUT = 30
+
+
+def get_fabric_api_base() -> str:
```

Comment:
this seems like a common argument. I'm not sure what It's for, but probably better move it to common?